conf/bk_server.conf

```diff
@@ -534,6 +534,12 @@ ledgerDirectories=/tmp/bk-data
 # True if the bookie should double check readMetadata prior to gc
 # verifyMetadataOnGC=false
 
+# True if bookie should persist entrylog file metadata and avoid in-memory object allocation
+gcEntryLogMetadataCacheEnabled=false
+
+# Directory to persist Entrylog metadata if gcPersistentEntrylogMetadataMapEnabled is true
+# gcEntryLogMetadataCachePath=
```

Comment:
we should document the behavior of this configuration parameter
in ServerConfiguration we are using a computed default value....
we should have a consistent and documented behavior

https://github.com/apache/bookkeeper/pull/1949/files#diff-ca6cec8ce70c1bb7a79d3aab8ce752deb12731a33348510c2f9472f5d8ef53eeR464

Reply:
good point. addressed it.